instructions.md

```diff
@@ -0,0 +1,140 @@
+# Kumo leasing instructions
+
+* There are 16 dell nodes available to be leased. Each node has 2 nics, em1 and em2.
+
+* The nodes are avaiable for short term leases of 6 hours, please talk to an admin
+if you want more time.
+
+* Nodes **will be** automatically powered down and taken away from your project once the lease expires.
+
+* For provisioning, we connect `em1` to the bmi provisioning network natively.
+
+* To connect your node to internet, connect any of your nics to the `internet` network
+either natively or as a tagged network. Making it tagged would require you to
+do some additional configuration on your node. Recommended way to do is to connect em2 to `internet` natively.
+If you know what you are doing, you can ignore this section.
+
+* If you require a public IP address for your node, talk to Naved/Rado.
+
+## Using HIL
+
```

Comment:
Writing few lines about HIL and what it is and what it does can help the users to know what they are doing.